packages/system-upgrade-controller/package.yaml

```diff
@@ -1,5 +1 @@
-url: https://github.com/rancher/system-charts.git
-subdirectory: charts/rancher-k3s-upgrader/0.7.0
-commit: 0e464d6f3afd709d8f3d9a528eb65973f696c5ae
-version: 104.0.0
-doNotRelease: false
+url: local
```

Comment:
I dropped it because the version will be read from the chart.yaml file in the local chart when it is not specified in the package.yaml file.

The file was updated to include the `version` for consistency.